product_docs/docs/jdbc_connector/42.3.2.1/01_jdbc_rel_notes/18_jdbc_42.3.2.1_rel_notes.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: "Version 42.3.2.1"
+
+---
+
+The EDB JDBC connector provides connectivity between a Java application and an Advanced Server database.
+
+New features, enhancements, bug fixes, and other changes in the EDB JDBC Connector `42.2.8.1` include:
+
+| Type           | Description                                                                     |
+| -------------- | ------------------------------------------------------------------------------- |
+| Upstream Merge | Merged with the upstream community driver version `42.2.8`. See the community [JDBC documentation](https://jdbc.postgresql.org/documentation/head/index.html) for details. |
+| Enhancement    | EDB JDBC Connector now supports EDB Postgres Advanced Server 12.                |
```

Comment:
Not sure what happened to the release notes, only the latest version is showing. If you look at the published version you'll see a set of older release notes.